agentMET4FOF/metrological_agents.py

```diff
@@ -198,3 +201,43 @@ def custom_plot_function(self, data, sender_agent, **kwargs):
         else:
             trace = go.Scatter()
         return trace
+
+class MetrologicalGeneratorAgent(MetrologicalAgent):
+    """An agent streaming a sine signal
+
+    Takes samples from an instance of :py:class:`MetrologicalSineGenerator` with sampling frequency `sfreq` and
+    signal frequency `sine_freq` and pushes them sample by sample to connected agents via its output channel.
```

Comment:
This part of the docstring reflects the default behaviour but actually not so well the generic behaviour of the class. It refers to the current default behaviour of the default parameter settings, but those might change in future. In particular the behaviour of `MetrologicalSineGenerator` might change or its parameter names (as has happend before), which would render this docstring as a lie, if we forget to change it accordingly, which I assume likely.

I would suggest to document the generic behaviour of our class and simply leave the rest to the documentation of the linked default class `MetrologicalSineGenerator` (see below).

```suggestion
    Takes samples from an instance of :py:class:`MetrologicalDataStreamMET4FOF
    <agentMET4FOF.metrological_streams.MetrologicalDataStreamMET4FOF>` and pushes them
    sample by sample to connected agents via its output channel.
```